set-web/src/main/java/eu/europeana/set/web/service/impl/UserSetServiceImpl.java

```diff
@@ -478,7 +478,9 @@ private List<String> reorderItemDescriptions(UserSet userSet, List<String> itemD
       // escape "/" to "\/" to match json string
       localId = StringUtils.replace(localId, "/", "\\/");
       for (String description : itemDescriptions) {
-        if (description.contains(localId)) {
+        String idWithoutSpace="\"id\":" + "\"" + localId + "\"";
```

Comment:
please remove the redundant concatenation "id":" should be only one string and the last quote should be concatenated as char '"'. Also the serilization on the search API side is consistent, it is allways with space